scripts/contactUs.js

```diff
@@ -14,13 +15,15 @@ function validate(){
 
 
     // Name can only contain letters a-z in lowercase
+    // Note
     function isValiduserName(inputText){
       var nameformat = /^[a-z]+$/;
       if(inputText.value.match(nameformat))
       {
         return true;
       }
       else {
+        // Note
```

Comment:
You're defining the `text` variable here and then immediately assigning it to the `errorMessage.innerHTML`.  You don't need to.  You can directly assign the error message to the `innerHTML` without the variable.